go-controller/pkg/util/net_linux.go

```diff
@@ -15,6 +15,74 @@ import (
 	utilnet "k8s.io/utils/net"
 )
 
+type NetLinkLibOperations interface {
+	LinkByName(ifaceName string) (netlink.Link, error)
+	LinkSetup(link netlink.Link) error
+	AddrList(link netlink.Link, family int) ([]netlink.Addr, error)
+	AddrDel(link netlink.Link, addr *netlink.Addr) error
+	AddrAdd(link netlink.Link, addr *netlink.Addr) error
+	RouteList(link netlink.Link, family int) ([]netlink.Route, error)
+	RouteDel(route *netlink.Route) error
+	RouteAdd(route *netlink.Route) error
+	RouteListFiltered(family int, filter *netlink.Route, filterMask uint64) ([]netlink.Route, error)
+	NeighAdd(neigh *netlink.Neigh) error
+	NeighList(linkIndex, family int) ([]netlink.Neigh, error)
+	ConntrackDeleteFilter(table netlink.ConntrackTableType, family netlink.InetFamily, filter netlink.CustomConntrackFilter) (uint, error)
+}
+
+type defaultNetLinkLibOper struct {
+}
+
+var netLinkOperations NetLinkLibOperations = &defaultNetLinkLibOper{}
```

Comment:
I dont think this is needed. Compiler will figure it out automatically.

Reply:
I assume you are saying that `var netLinkOperations NetLinkLibOperations` is sufficient, right?

Reply:
I followed a similar instantiation per @girishmg suggestion at https://github.com/ovn-org/ovn-kubernetes/pull/1371#discussion_r436113148 for a different PR. Will keep the change as-is as its not an issue/bug. Thanks